build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Pex isn't quite as flexible as I'd hoped.

My main gripe is that you can't tell it to bundle a package from a local path, or a git repos. It seems only capable of pulling packages from PyPi.

It does support using a custom PyPi server, so we could look in to that at some point. In the meantime, we will have to push to PyPi before we can build a new agent for testing.